wiki/Contests/KCC/2/ko.md

```diff
@@ -0,0 +1,95 @@
+---
+tags:
+  - KCC2022
+  - KCC 2022
+---
+
+# Korean Charting Contest 2022
+
+![KCC2022 배너](img/banner.png)
+
+**Korean Charting Contest 2022** (***KCC2022***) 은 한국인을 위한 osu!mania 매핑 대회입니다. 이 대회는 두 번째 대회입니다.
+
+## 대회 일정
+
+| 일정 | 날짜 (UTC+9) |
+| --: | :-- |
+| 제출 기간 | 2022-01-30/2022-02-27 |
+| 심사 기간 | 2022-02-28/2022-03-13 |
+| 결과 발표 | 2022-03-14 |
+
+## 상품
+
+| 순위 | 상품 |
+| :-: | :-- |
+| ![금상](/wiki/shared/crown-gold.png "1위") | 50,000 KRW, 프로필 뱃지 (미정) |
+| ![은상](/wiki/shared/crown-silver.png "2위") | 20,000 KRW |
+| ![동상](/wiki/shared/crown-bronze.png "3위") | 10,000 KRW |
+
+![KCC2022 뱃지](img/badge.png)
+
+## 주최
+
+| 직책 | 인원 |
+| :-- | :-- |
+| 주최자 | ![][flag_KR] [Garalulu](https://osu.ppy.sh/users/757783) |
+| 매니저 | ![][flag_KR] [Oni Suika](https://osu.ppy.sh/users/4848023) |
+| 디자이너 | ![][flag_KR] [Manwon](https://osu.ppy.sh/users/4129020) |
+| 심사위원 | ![][flag_CN] [_Stan](https://osu.ppy.sh/users/1653229), ![][flag_SG] [Evening](https://osu.ppy.sh/users/2193881), ![][flag_KR] [Garalulu](https://osu.ppy.sh/users/757783), ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573), ![][flag_KR] [Remuring](https://osu.ppy.sh/users/6522146) |
+
+## 링크
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1507792)
+
+## 곡 리스트
+
+### Instrumental
+
+| Song | Length | BPM |
```

Comment:
shouldn't these be translated? maybe you accidentally copied the english version instead of libneko's review

Reply:
at first I don't think it should be translated too but after talking with libneko, I changed my mind. fixed!